sdks/java/extensions/sql/src/test/java/org/apache/beam/sdk/extensions/sql/meta/provider/kafka/KafkaTableProviderIT.java

```diff
@@ -210,7 +210,7 @@ public void testFakeNested() throws InterruptedException {
     String createTableString =
         String.format(
             "CREATE EXTERNAL TABLE kafka_table(\n"
-                + "headers ARRAY<ROW<key VARCHAR, values ARRAY<BYTES>>>,"
+                + "headers ARRAY<ROW<key VARCHAR, `values` ARRAY<VARBINARY>>>,"
```

Comment:
Is this still necessary after https://github.com/apache/beam/pull/14205?

Reply:
Nope. Rebased